src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -0,0 +1,49 @@
+# # -*- coding: utf-8 -*-
+
+from ..exceptions import DashboardException
+from ..model.feedback import Feedback
+from ..rest_client import RequestException
+from ..security import Scope
+from ..services import feedback
+from . import ApiController, ControllerDoc, RESTController
+
+
+@ApiController('/feedback', Scope.CONFIG_OPT)
+@ControllerDoc("Feedback API", "Report")
+class FeedbackController(RESTController):
+    issueAPIkey = None
+
+    def __init__(self):  # pragma: no cover
+        super(FeedbackController, self).__init__()
+        self.cephTrackerClient = feedback.CephTrackerClient()
+
+    @RESTController.MethodMap(version='0.1')
+    def create(self, project, tracker, subject, description):
+        """
+        Create an issue.
+        :param project: The buggy ceph component.
```

Comment:
Is "buggy" the appropriate term here? If it's a feature or enhancement request, there might not actually be a bug. How about "affected" instead?